presto-product-tests/bin/run_on_docker.sh

```diff
@@ -189,6 +190,11 @@ if [[ "$ENVIRONMENT" == "multinode-tls" ]]; then
     CLI_ARGUMENTS="--server https://presto-master.docker.cluster:7778 --keystore-path /docker/volumes/conf/presto/etc/docker.cluster.jks --keystore-password 123456"
 fi
 
+TLS_CERTIFICATE=""
```

Comment:
Do you think it is possible to merge https://github.com/prestodb/presto/pull/7725 before this PR? So we don't need to add this workaround.